shotover-proxy/example-configs/redis-cluster-handling/topology.yaml

```diff
@@ -0,0 +1,15 @@
+---
+sources:
+  redis_prod:
+    # define how shotover listens for incoming connections from our client application (`redis-benchmark`).
+    Redis:
+      listen_addr: "127.0.0.1:6379"
+chain_config:
+  redis_chain:
+    # configure Shotover to connect to the Redis cluster via the shotover instances passthrough node
+    - RedisSinkCluster:
+        destination:
+          Handling: "127.0.0.1:2220"
```

Comment:
As above... I think destination (or something like it) needs to be the contact point keyword and Handling / Hiding modes defined above. Or even an explicity `mode: Handling` property.

I know this makes the rust side less ergonomic, but right now the user configuration feels confusing with the way it's set up.